src/mapgen.cpp

```diff
@@ -1928,6 +1929,8 @@ void mapgen_function_json::generate( map *m, const oter_id &terrain_type, const
         lua_mapgen( m, terrain_type, md, t, d, luascript );
     }
 
+    place_stairs( m, terrain_type, md );
```

Comment:
Should place_stairs go 4 lines lower, after the call to m->rotate(...) ?

Seems like we should want it to be one of the last things to happen in json creation, both as a general precaution and specifically so that ALIGN_STAIRS (if on) is trying to align with the final orientation, not pre-rotation.

Reply:
Could be true, but I am not sure - stairs code in house function was located prior to rotation before I have moved it to a separate method.